src/doc/src/reference/source-replacement.md

```diff
@@ -131,3 +141,11 @@ is placed on the name of each directory.
 Each crate in a directory source also has an associated metadata file indicating
 the checksum of each file in the crate to protect against accidental
 modifications.
+
+## Git sources
+
+Git sources represent repositories used by [git-based dependencies]. They're
+used by `cargo vendor` to replace git-based dependencies with local directories.
```

Comment:
The comment about `cargo vendor` isn't making sense to me, on its own or how it fits into the overall context of this section.